### PR TITLE
Mobile menu deployment

### PR DESCRIPTION
**Pull request by inakiesk**
This branch is adding a menu once you click on the menu icon on the toolbar section and the branch has the following features added:
-Added JS file with appropriate code to be able to deploy the menu.
-Added X image in order to close the menu options.
-Added menu options to display once menu icon is clicked.

The project specifications require the following:
In mobile, implement the following features:

    -When the user clicks (or taps) the hamburger button, the mobile menu appears. **-Menu appears as requested**
    -When the user clicks (or taps) the close (X) button, the mobile menu disappears. **-Menu disappears as requested.**
    -When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears. **-Menu disappears as requested**
    -When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. **-Options send user to correct section**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -15,6 +15,28 @@ body {
...
+  list-style-type: none;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #344563;
+  text-decoration: none;
+}
+
```
Comment (sumairq) [inline style.css:39]: - [ ] Good job implementing the JavaScript functionality however, you design for mobile menu needs to be similar to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) template . Kindly implement the design before submitting your project for another review. cheers 👍 

The design should look like this             |  Your Design
:-------------------------:|:-------------------------:
My Portfolio  |  Inaki's Portfolio_ Web developer

**Review by sumairq — CHANGES_REQUESTED** [review]
### Hi @inakiesk , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by inakiesk** [comment]
I completely forgot to check the Figma design when doing my menu! Sorry!! The design is fixed now!
Alt Text

Summary of new changes:
-Changed the menu design to fit Figma template.
-Fixed an error that caused menu to close when you clicked on the background.
-Fixed a new linter error that appeared while fixing everything else.

File: style.css [hunk 2]
```diff
@@ -15,12 +15,53 @@ body {
...
+#menuBackground {
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  background-color: #6070ffde;
+  background-size: cover;
+  top: 0%;
+  margin-top: 0;
+  z-index: 50;
+}
```
Comment (Reem-lab) [inline style.css:31]: - [ ] As the previous code reviewer aske  I think here you need to blur the background to match the Figma design as much as possible ✅✅

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi Team , 👋🏻👋🏻

Good job so far!🔥🔥🔥💪🏻💪🏻
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- No Linters error ✅✅
- Clean code ✅✅
- Great working on the Functionality ✅✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by inakiesk** [comment]
Summary of new changes:
-Fixed background color to fit figma design.
-Applied a backdrop filter blur to match Figma design (property not compatible with some firefox versions).

**Pull request merged**